recipes/phonenumbers/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "7.4.2" %}
+package:
+  name: phonenumbers
+  version: {{ version }}
+
+source:
+  fn: phonenumbers-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/phonenumbers/phonenumbers-{{ version }}.tar.gz
+  md5: 2737b6e373e8e4ee1617a58803174473
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - phonenumbers
+    - phonenumbers.carrierdata
+    - phonenumbers.data
+    - phonenumbers.geodata
+    - phonenumbers.shortdata
+    - phonenumbers.tzdata
+
+about:
+  home: https://github.com/daviddrysdale/python-phonenumbers
+  license: Apache 2.0
+  summary: "Python version of Google's common library for parsing, formatting, storing and validating international phone numbers."
```

Comment:
Could swap
> Python version of Google's common library
for
> Python port of Google's library

Could also drop "international".  What's the optimal length?